### PR TITLE
Updating iiblib to reflect latest changes in IIB API

### PR DESCRIPTION
**Pull request by lipoja**
Updating parameters that are send, correct endpoints and updating structure of received response from API

### CODE REVIEW

File: iiblib/iibclient.py [hunk 1]
```diff
@@ -144,7 +157,7 @@ def next(self):
     def prev(self):
         """Load items for previous page and set it as current"""
 
-        if self.page > 0:
+        if self.page > 1:
```
Comment (midnightercz) [inline iiblib/iibclient.py]: IIB starts paging from 1?

File: requirements.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-requests
+requests>=2.4.2
```
Comment (midnightercz) [inline requirements.txt]: what's missing in requests bellow 2.4.2

File: iiblib/iibclient.py (hunk 1 above)
Comment (lipoja) [reply iiblib/iibclient.py]: According to documentation of flask sqlalchemy pages are 1 indexed
https://flask-sqlalchemy.palletsprojects.com/en/2.x/api/#flask_sqlalchemy.Pagination.page

File: requirements.txt (hunk 2 above)
Comment (lipoja) [reply requirements.txt]: parameter "json" for get and post ... you've used data which was is no correctly converted to content type application/json. Therefore I've used "json=" and it was introduced in 2.4.2

**Pull request merged**